.github/workflows/dependabot-changesets.yml

```diff
@@ -0,0 +1,15 @@
+name: "Dependabot Changesets"
+
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  changesets:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'sigstore' &&  github.actor == 'dependabot[bot]' }}
```

Comment:
What is the `github.repository_owner == 'sigstore'` protecting here?

Reply:
> What is the `github.repository_owner == 'sigstore'` protecting here?

So it doesn't run on forks but dunno if really worth it. 

Yeah maybe could also provide a reusable workflow that invokes itself?

Reply:
or invoke the reusable workflow from the action? 😄